### PR TITLE
Math Magician: React rounter

### PR DESCRIPTION
**Pull request by Strangeal**
### Full website
So far it was enjoyable😊 implementing all the necessary requirements for the project I learned so many things and it was also fun working on this project, what I did in the whole project :

- Added 3 pages: Home, Calculator, and Quote
- Added styles to improve the look and feel
- Created a route for every page
- Navigation links for all 3 routes are displayed on each page
- Used React router to make routes works

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -9,9 +13,14 @@ class App extends Component {
 
   render() {
     return (
-      <div className="App">
-        <Calculate />
-      </div>
+      <>
+        <Navbar />
+        <Routes>
+          <Route path="/math_magician" element={<Home />} />
```
Comment (omar-muhamad) [inline src/App.js]: - [x] The `Home` page suppose to be the root route, So please make this path the root path like that:
```javascript
<Route path="/" element={<Home />} />
```

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import '../App.css';
+import { Link, NavLink } from 'react-router-dom';
+
+const Navbar = () => {
+  const links = [
+    {
+      id: 1,
+      path: '/math_magician',
```
Comment (omar-muhamad) [inline src/components/Navbar.js]: - [x] Same here, Please make this path the root path like that:
```javascript
path: '/',
```

File: src/components/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        <ul className="nav__menu">
+          {links.map((link) => (
+            <li key={link.id} className="nav__item">
+              {' '}
+              |
+              <NavLink to={link.path} className={({ isActive }) => `link ${(isActive ? 'active' : '')}`} end>
+                {link.text}
+              </NavLink>
+            </li>
+          ))}
```
Comment (omar-muhamad) [inline src/components/Navbar.js:38]: - [x] Great job here using conditional rendering 👍.

File: src/components/Calculator.js [hunk 4]
```diff
@@ -28,19 +28,23 @@ export const Calculator = () => {
...
+          <div className="screen">{next || total || 0}</div>
+          <div className="calculator__keys">
+            {buttonKeys.map((each) => (
+              <button
+                key={each}
+                onClick={(e) => onClickHandler(e)}
+                type="button"
+              >
+                <span className="front">{each}</span>
+              </button>
```
Comment (omar-muhamad) [inline src/components/Calculator.js:45]: - [x] Great job here using conditional rendering 👍.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Strangeal,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Strangeal, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**